File: android/src/main/java/com/jjlf/rngradient/RadialGradientView.java
package com.jjlf.rngradient;

import android.content.Context;
import android.view.ViewGroup;

public class RadialGradientView extends ViewGroup {

    private final GradientLayer mDrawable = new GradientLayer();
    RadialGradientView(Context context){
        super(context);
        setBackground(mDrawable);
        setClipChildren(false);
        mDrawable.setType(GradientLayer.RADIAL);
    }

    GradientLayer getDrawable() {
        return mDrawable;
    }
    @Override
    protected void onLayout(boolean changed, int l, int t, int r, int b) {

    }
}

File: android/src/main/java/com/jjlf/rngradient/RadialGradientView.kt
package com.jjlf.rngradient

import android.content.Context
import android.view.ViewGroup



class RadialGradientView(context: Context): ViewGroup(context) {


    private val mDrawable = GradientLayer()
    init{
        background = mDrawable
        clipChildren = false
        mDrawable.setType(GradientLayer.RADIAL)
    }

    fun getDrawable(): GradientLayer {
        return mDrawable
    }

    override fun onLayout(changed: Boolean, l: Int, t: Int, r: Int, b: Int) {

    }

}